testplan/report/testing/schemas.py

```diff
@@ -67,14 +77,132 @@ def _deserialize(self, value, attr, data):
         })
 
 
+class EntriesField(fields.Field):
+    """
+    Handle encoding problems gracefully
+    """
+
+    _BYTES_KEY = 'bytes'
+
+    @staticmethod
+    def _binary_to_base64(binary_obj):
+        return base64.standard_b64encode(binary_obj).decode('us_ascii')
```

Comment:
So it says [here](https://docs.python.org/3/library/codecs.html#standard-encodings) "Optimization opportunity recognized for us-ascii" which isn't super clear but could only be a good thing, but from [here](https://github.com/python/cpython/blob/5641b6d3115f1c2700d2f84216710d3778a23d02/Lib/encodings/aliases.py#L34) and [here](https://github.com/python/cpython/blob/5641b6d3115f1c2700d2f84216710d3778a23d02/Objects/unicodeobject.c#L3212) it does seem like "us_ascii" is exactly equivalent to "ascii". Though I don't know why they would they specify "us-ascii" in the docs instead of just "ascii" unless the "us" bit means something special.

Reply:
from the first doc you linked I'm pretty sure "us_ascii" is just an alias for "ascii". The "a" in ascii does stand for "American" after all, it would be weird to have some special US variant of that.